### PR TITLE
Add GitHub Dark Mode

### PR DESCRIPTION
**Pull request by MrWillCom**
This pull request added my browser extension [GitHub Dark Mode](https://github.com/MrWillCom/github-dark-mode).

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -711,5 +711,17 @@
     "stars": 3888,
     "installCount": 100000,
     "lastUpdate": "30 Jul 2020"
+  },
+  {
+    "name": "GitHub Dark Mode",
+    "source": "https://github.com/MrWillCom/github-dark-mode",
+    "tags": [
+      "dark-mode"
```
Comment (stefanbuck) [inline data.json:719]: Values define in the tags fields appeare as filter in the web version. Therefore I would prefer to rename this value to `theme`

**Review by stefanbuck — APPROVED** [review]
Apart from one minor suggestions this looks good to me.

Out of interest, are you considering publishing it as Chrome extension as well?

File: data.json (hunk 1 above)
Comment (MrWillCom) [reply data.json]: Thanks, `theme` is really a better choice.

Commt https://github.com/stefanbuck/awesome-browser-extensions-for-github/pull/148/commits/5d83992726ab9ac6ccae4ded9be09f7773a00814 changed this.

**Comment by stefanbuck** [comment]
Thanks for submitting and of course for creating this extension

**Comment by MrWillCom** [comment]
> Thanks for submitting and of course for creating this extension

This is a good chance to let more people know my extension. Thank you for merging my pull request. :smile:

**Comment by stefanbuck** [comment]
I'm not a dark mode user myself, but I'm pretty sure many people will love it.

**Pull request merged**